dpnp/backend/extensions/lapack/heevd.cpp

```diff
@@ -228,6 +228,130 @@ std::pair<sycl::event, sycl::event>
     return std::make_pair(args_ev, heevd_ev);
 }
 
+std::pair<sycl::event, sycl::event>
+    heevd_batch(sycl::queue exec_q,
```

Comment:
```suggestion
    heevd_batch(sycl::queue &exec_q,
```

Reply:
We should pass _sycl::queue_ by references ensures no copy
This is a useful comment. Thanks!
I will fix for **syevd,** **syevd_batch**, **heevd** and **heevd_batch** here and create a new PR to change this for all linear functions